packages/remix-react/components.tsx

```diff
@@ -568,12 +571,11 @@ function V1Meta() {
     if (routeModule.meta) {
       let routeMeta =
         typeof routeModule.meta === "function"
-          ? routeModule.meta({
+          ? (routeModule.meta as V1_MetaFunction)({
```

Comment:
shouldn't this type already come from the routeModule type? we don't have to cast the other types of this object, why is this property special?

Reply:
Because it's a union type on the route module, TS doesn't know how to disambiguate at the call-site.